api/operator_ent_test.go

```diff
@@ -0,0 +1,28 @@
+//go:build ent
+// +build ent
+
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestOperator_LicenseGet(t *testing.T) {
+	t.Parallel()
+	c, s, _ := makeACLClient(t, nil, nil)
+	defer s.Stop()
+
+	operator := c.Operator()
+
+	// Make atuhenticated request.
```

Comment:
```suggestion
	// Make authenticated request.
```

(and "unauthenticated", below)